src/app.service.ts

```diff
@@ -60,7 +52,7 @@ export class AppService {
   }
 
   /**
-   * returns all log messages created
+   * returns all created log messages during runtime of service
```

Comment:
gibt es die erzeugten nachrichten während der laufzeit zurück (d.h. "during runtime" beschreibt den zeitpunkt der rückgabe), oder gibt es die während der laufzeit erzeugten nachrichten zurück (d.h. "during runtime" beschreibt den zeitpunkt, zu dem die nachricht erzeugt  wurde)?
falls letzteres gemeint ist, wäre folgende formulierung eindeutiger:
```suggestion
   * returns all log messages created during runtime of service
```